internal/helper/sink_ack_ordering_test.go

```diff
@@ -45,7 +45,7 @@ func TestAckOrderingSink(t *testing.T) {
 	cancel()
 
 	err := eg.Wait()
-	assert.NoError(err)
+	assert.EqualError(err, "context canceled")
```

Comment:
It would be nicer to just compare the error to `context.Cancelled` 
```suggestion
	assert.Equal(context.Cancelled, err)
```